src/subscript/sector2fluxnum/sector2fluxnum.py

```diff
@@ -0,0 +1,251 @@
+
+    # ###########################################
+
+    print("Writing DUMPFLUX DATA-file ...")
+    new_data_file = do.Datafile("%s.DATA" % args.ECLIPSE_CASE[0])
+
+    if new_data_file.has_KW("DUMPFLUX") or new_data_file.has_KW("USEFLUX"):
+        raise Exception("ERROR: FLUX keywords already present in input ECL_CASE")
+
+    # ###########################################
+
+    new_data_file.create_DUMPFLUX_file(FLUXNUM_filename)
+
+    # ###########################################
+
+    if args.test:
+        args.test = os.path.abspath(args.test).split(".")[0:-1]
+
+        if not os.path.isfile("%s.FLUX" % args.test[0]):
+            raise Exception("ERROR: FLUX file from DUMPFLUX run not created")
+
+        # Needs the coordinates from the
+        print("Generating new FLUX file...")
+
+        grid_coarse = EclGrid("%s.EGRID" % args.test[0])
+        grid_fine = EclGrid("%s.EGRID" % args.test[0])
+        flux_fine = EclFile("%s.FLUX" % args.test[0])
+
+    else:
+        print("Executing DUMPFLUX NOSIM run ...")
+        if args.version:
+            new_data_file.run_DUMPFLUX_nosim(args.version)
+        else:
+            new_data_file.run_DUMPFLUX_nosim()
+
+        if not os.path.isfile("DUMPFLUX_%s.FLUX" % ECLIPSE_CASE_ROOT):
+            raise Exception("ERROR: FLUX file from DUMPFLUX run not created")
+
+        # Needs the coordinates from the
+        print("Generating new FLUX file...")
+
+        grid_coarse = EclGrid("DUMPFLUX_%s.EGRID" % ECLIPSE_CASE_ROOT)
+        grid_fine = EclGrid("DUMPFLUX_%s.EGRID" % ECLIPSE_CASE_ROOT)
+        flux_fine = EclFile("DUMPFLUX_%s.FLUX" % ECLIPSE_CASE_ROOT)
+
+    # Reads restart file
+    if args.restart:
+        rst_coarse = EclFile("%s.UNRST" % args.restart[0])
+    else:
+        rst_coarse = EclFile("%s.UNRST" % args.ECLIPSE_CASE[0])
+
+    flux_object_fine = ffo.Fluxfile(grid_fine, flux_fine)
+
+    # ######################################################
+    # Creating map
+    # ######################################################
+    f_c_map = ffo.create_map_rst(
+        flux_object_fine, grid_coarse, scale_i=1, scale_j=1, scale_k=1
+    )
+
+    # ######################################################
+    # Importing elements
+    # ######################################################
+
+    # Open FortIO stream
+    fortio = FortIO("%s.FLUX" % args.OUTPUT_CASE[0], mode=FortIO.WRITE_MODE)
+
+    ffo.write_new_fluxfile_from_rst(
+        flux_object_fine, grid_coarse, rst_coarse, f_c_map, fortio
+    )
+
+    # Close FortIO stream
+    fortio.close()
+
+    # ######################################################
+    # Writing USEFLUX suggestion
+    # ######################################################
+
+    print("Writing suggestion for USEFLUX DATA-file ...")
+    new_data_file.create_USEFLUX_file(FLUXNUM_filename, args.OUTPUT_CASE[0])
+    new_data_file.set_USEFLUX_header(args)
+
+
+def main():
+    """
+    main method
+    """
+    header_print()
```

Comment:
I wonder if this function can be skipped altogether. The QC reminder is not correct to have there, and the information about the script is available through `sector2fluxnum --help`.